### PR TITLE
EL: randomUUID to String

### PR DESCRIPTION
**Pull request by Amerousful**

### CODE REVIEW

**Comment by Isammoc** [comment]
@Amerousful What about documentation?

**Comment by Amerousful** [comment]
@Isammoc Thx for this point. Can you check is it everything ok?

File: src/docs/content/reference/current/core/session/el/index.md [hunk 1]
```diff
@@ -83,9 +83,11 @@ Gatling EL provide the following built-in functions:
 "#{foo.htmlUnescape()}"
 
 // generate a random UUID (fast but cryptographically insecure)
+// UUID will be provided as String
```
Comment (slandelle) [inline src/docs/content/reference/current/core/session/el/index.md:86]: `generate a random UUID String` shorter and clearer :)

File: src/docs/content/reference/current/core/session/el/index.md [hunk 2]
```diff
@@ -83,9 +83,11 @@ Gatling EL provide the following built-in functions:
 "#{foo.htmlUnescape()}"
 
 // generate a random UUID (fast but cryptographically insecure)
+// UUID will be provided as String        
 "#{randomUuid()}"
 
 // generate a random UUID (standard java.util.UUID#randomUUID: cryptographically secure, but slower)
+// UUID will be provided as String
```
Comment (slandelle) [inline src/docs/content/reference/current/core/session/el/index.md:90]: same as above

**Review by slandelle — CHANGES_REQUESTED** [review]
Please also improve commit message, eg:

```
Make `randomUuid()` and `randomSecureUuid()` Gatling EL functions generate Strings and not java.util.UUID, close [#4369](https://github.com/gatling/gatling-private/issues/4369)

**Review by Isammoc — COMMENTED** [review]
1. This should requires only one commit
2. @slandelle Does it break binary compatibility (other than this obvious `randomUuid`)?

**Comment by slandelle** [comment]
> This should requires only one commit

Yes, we will squash.

> @slandelle Does it break binary compatibility (other than this obvious randomUuid)?

No, it doesn't. ElCompiler is not directly exposed.
There's indeed a behavior change, but not binary incompatibility, that's why I aimed at 3.9.1 and not 3.10.0.

**Review by slandelle — APPROVED** [review]

**Comment by slandelle** [comment]
cheery-picked and squashed.

Thanks a lot!

**Pull request closed without merging**